Realm/RLMSyncUser.h

```diff
@@ -38,17 +38,20 @@ typedef void(^RLMUserCompletionBlock)(RLMSyncUser * _Nullable, NSError * _Nullab
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- A `RLMSyncUser` instance represents a single Realm Object Server user account (or just user).
+ A `RLMSyncUser` instance represents a single Realm Object Server user account
+ (or just user).
```

Comment:
I don't get what the parentheses mean. How is "just user" a different thing than what's stated before ("a single Realm Object Server user account")?